Raw json:

{'hasError': True, 'errParameter': ['alluxio.user.network.rpc.max.inbound.message.size'], 'reason': ['The value of alluxio.user.network.rpc.max.inbound.message.size is 200MB, which is not a valid value. The value must be a positive integer.']}

{'hasError': True, 'errParameter': ['alluxio.user.network.rpc.max.inbound.message.size'], 'reason': ['The value of alluxio.user.network.rpc.max.inbound.message.size is too large. The maximum value is 100MB.']}

{'hasError': True, 'errParameter': ['alluxio.user.network.rpc.max.inbound.message.size', 'alluxio.user.streaming.reader.close.timeout', 'alluxio.user.client.cache.timeout.threads'], 'reason': ['The value of alluxio.user.network.rpc.max.inbound.message.size is not a valid number.', 'The value of alluxio.user.streaming.reader.close.timeout is not a valid duration.', 'The value of alluxio.user.client.cache.timeout.threads is not a valid number.']}

Final result:

There are 1 misconfiguration parameters in the input: alluxio.user.network.rpc.max.inbound.message.size
[Ciri] Reason for alluxio.user.network.rpc.max.inbound.message.size: The value of alluxio.user.network.rpc.max.inbound.message.size is 200MB, which is not a valid value. The value must be a positive integer.
